src/zarr/testing/strategies.py

```diff
@@ -218,10 +219,14 @@ def key_ranges(
         [(key, (range_start, range_end)),
          (key, (range_start, range_end)),...]
     """
+
+    def make_request(start: int, length: int) -> RangeByteRequest:
+        return RangeByteRequest(start, end=min(start + length, max_size))
```

Comment:
This also isn't showing up in code coverage.

Reply:
that's a bad report, or perhaps out-of-date. this PR also enables coverage recording for the hypothesis tests.